Sources/TokamakCore/CGStubs.swift

```diff
@@ -15,7 +15,11 @@
 //  Created by Max Desiatov on 08/04/2020.
 //
 
+#if canImport(Glibc)
```

Comment:
Is there anything in this file that requires Foundation?

Reply:
No, but Xcode doesn’t like the Glibc import and Foundation includes the necessary trig functions. Is there something else that should be imported in native platforms?

Reply:
It doesn’t seem to affect the warm bundle size at all.

Reply:
`import Darwin` for the trig functions.

Reply:
I'd suggest importing based on the platform then:

```suggestion
#if os(WASI)
```